channels/tsmf/client/alsa/tsmf_alsa.c

```diff
@@ -226,14 +226,15 @@ static void tsmf_alsa_free(ITSMFAudioDevice* audio)
 
 ITSMFAudioDevice* alsa_freerdp_tsmf_client_audio_subsystem_entry(void)
 {
-	TSMFAlsaAudioDevice* alsa;
-	alsa = (TSMFAlsaAudioDevice*)malloc(sizeof(TSMFAlsaAudioDevice));
-	ZeroMemory(alsa, sizeof(TSMFAlsaAudioDevice));
+	TSMFAlsaAudioDevice* alsa = (TSMFAlsaAudioDevice*)calloc(1, sizeof(TSMFAlsaAudioDevice));
```

Comment:
This cast should not be needed (the `(TSMFAlsaAudioDevice*)`), applies to the other `calloc`s here in this MR too.